app/Http/Controllers/CalendarController.php

```diff
@@ -13,11 +14,15 @@ public function __invoke(Request $request): Response
     {
         $calendar = Calendar::create()
             ->name('Larastreamers')
-            ->description('There is no better way to learn than by watching other developers code live. Find out who is streaming next in the Laravel world.');
+            ->description('There is no better way to learn than by watching other developers code live. Find out who is streaming next in the Laravel world.')
+            ->refreshInterval(CarbonInterval::hour()->totalMinutes)
+            ->productIdentifier('larastreamers.com');
 
-        Stream::upcoming()->each(fn(Stream $stream) => $calendar->event(
-            $stream->toCalendarItem()
-        ));
+        Stream::query()
+            ->where('scheduled_start_time', '>=', now()->subYear()->startOfYear())
```

Comment:
Can you add a scope for this to make it more obvious like `notOlderThanAYear` or similar?